api/filter/http/gzip.proto

```diff
@@ -22,18 +24,21 @@ message Gzip {
   // compression. Default value is 30.
   google.protobuf.UInt32Value content_length = 2 [(validate.rules).uint32.gte = 30];
 
-  enum CompressionLevel {
-    DEFAULT = 0;
-    BEST = 1;
-    SPEED = 2;
+  message CompressionLevelEnum {
```

Comment:
nit: I would call the message `CompressionLevel` and the inner enum `Enum` or `CompressionLevelEnum` because then the values can be accessed in code via `CompressionLevel::DEFAULT`, etc.

Reply:
Done